pp_netlib/network.py

```diff
@@ -242,15 +260,16 @@ def load_network(self, network_file):
                 self.graph = self.nx.read_graphml(network_file)
                 num_nodes = len(self.graph.nodes())
                 num_edges = len(self.graph.edges())
-            sys.stderr.write(f"Loaded network with {num_nodes} nodes and {num_edges} edges.\n")
+        
+            sys.stderr.write(f"Loaded network with {num_nodes} nodes and {num_edges} edges with {self.backend}.\n\n")
 
         # useful for cugraph, to be added in later
         # elif file_extension in [".csv", ".tsv", ".txt"]:
         #     sys.stderr.write("The network file appears to be in tabular format, please load it as a dataframe and use the construct method to build a graph.\n")
         #     sys.exit(1)
 
         else:
-            sys.stderr.write("File format not recognised.")
+            sys.stderr.write("File format not recognised. Quitting...\n\n")
```

Comment:
Better to raise a RuntimeError again, then the quit/fail will be obvious

Reply:
done